test/users.spec.js

```diff
@@ -117,4 +121,29 @@ describe('Authentication', () => {
         });
     });
   });
+
+  describe('GET /api/users?token', () => {
+    const baseUrl = '/api/users/verify';
+    it('should return true if a user is verified', done => {
+      chai
+        .request(server)
+        .get(`${baseUrl}?token=${generateToken(tokenPayload)}`)
+        .end((err, res) => {
+          expect(res).to.have.status(200);
+          expect(res.body.verified).to.equal(true);
+          done(err);
+        });
+    });
+
+    it('should check that the token is valid', done => {
+      chai
+        .request(server)
+        .get(`${baseUrl}?token=io.fjf.tir`)
+        .end((err, res) => {
+          expect(res).to.have.status(500);
+          expect(res.body.verified).to.equal(undefined);
```

Comment:
Seconding this

Reply:
Is the user not expecting an error message?

Reply:
@blackdevelopa  ☝️ 
@LordUche  It is `undefined` because the `verified` property gets tacked on to the response body only when verification succeeds.
I also agree that we need to approach testing more verbosely......  if it's being sent out in the response, test it.

Reply:
Why would the `verified` property ever be `undefined`? If verification fails then the answer to whether or not the link was `verified` is `false`, no?

Reply:
The latest commit updates the test and addresses this